examples/pl_basics/README.md

```diff
@@ -8,13 +8,19 @@ Here are 5 MNIST examples showing you how to gradually convert from pure PyTorch
 
 The transition through [LightningLite](https://pytorch-lightning.readthedocs.io/en/stable/starter/lightning_lite.html) from pure PyTorch is optional but it might be helpful to learn about it.
 
+Please add `PYTHONPATH` as follows before executing the following command.
+
+```bash
+export PYTHONPATH="../../../lightning/src:$PYTHONPATH"
+```
+
```

Comment:
I would suggest user to install the package then mess up with python paths

Reply:
I get an error without the above code or the following `sys.path.insert`. How do I actually explain it to the user in README.md?

```python
import sys
sys.path.insert(0,'../../../lightning/src')
```

The following is the error message for `ModuleNotFoundError`.

```
⋊> ~/l/c/l/e/convert_from_pt_to_pl on master ⨯ python image_classifier_1_pytorch.py
Traceback (most recent call last):
  File "image_classifier_1_pytorch.py", line 28, in <module>
    from pytorch_lightning.demos.boring_classes import Net
ModuleNotFoundError: No module named 'pytorch_lightning.demos'
⋊> ~/l/c/l/e/convert_from_pt_to_pl on master ⨯ 
```

Reply:
You're seeing the error because you're running the example with one of the released `pytorch-lightning` versions, I think. `pytorch_lightning.demos` hasn't been released yet and is only available in the main branch `master` at the moment.

Reply:
I understand what to do thanks to the notes. I'm sure there is no problem with that comment.

By the way, even if the example of the repository and the version of the package are the same in 1.6.4, an error will occur, but since it has already been released, I think that it does not need to be fixed unless it is a big problem. I think I should fix it in the future.